changelogs/fragments/311-add-defaults-fields-snmp-community.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_modify, api_info - add ``default`` field to ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/311).
```

Maybe in that case also add:
```suggestion
  - api_modify, api_info - add read-only ``default`` field to ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/311).
```